src/sidebar/cross-origin-rpc.js

```diff
@@ -11,7 +25,19 @@ let preStartQueue = [];
  */
 const registeredMethods = store => {
   return {
-    changeFocusModeUser: store.changeFocusModeUser,
+    /**
+     * @param {FocusUserInfo} userInfo
+     */
+    changeFocusModeUser: userInfo => {
+      store.changeFocusModeUser(userInfo);
+
+      const groupIds = userInfo?.groups ?? [];
+      const filteredGroupIds = normalizeGroupIds(groupIds, store.allGroups());
```

Comment:
In this approach, all identifiers for groups, be they `groupids` or `ids`, are normalized to `id`s before passing them to the store layer.